crates/samples/echomain-stateful/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "samples_echomain_stateful"
+version = "0.0.1"
+edition = "2018"
+
+[dependencies]
+log = "0.4"
+env_logger = "*"
+ctrlc = { version = "3.0", features = ["termination"] }
+tokio = { version = "1", features = ["full"] }
+windows-core = "0.51"
+
+[dependencies.windows]
+version = "0.51"
+features = [
+    "implement"
+]
+
+[dependencies.fabric_base]
+path = "../../fabric/base"
+features = [
+    "implement",
+    "ServiceFabric",
+    "ServiceFabric_FabricCommon",
+    "ServiceFabric_FabricCommon_FabricRuntime"
+]
+
+[dependencies.fabric_rs]
```

Comment:
If you want to demo COM usage you should ideally remove this dependency

Reply:
I can see even echomain is using use fabric_rs::{AsyncContext, StringResult}; ... seems like there are some common namespaces